### PR TITLE
Update CoinTickersFetcher with removing creating AlphaWallet.Address for Ticker as its expensive operation #3881

### LINKED ISSUE #3881: Update CoinTickersFetcher with removing creating AlphaWallet.Address for `Ticker` as its expensive operation

### PR DESCRIPTION
**Pull request by oa-s**
Closes #3881

### CODE REVIEW

File: AlphaWallet/Core/CoinTicker/CoinTickersFetcher.swift [hunk 1]
```diff
@@ -277,8 +277,26 @@ fileprivate struct Ticker: Codable {
...
+
         guard isServerSupported(tokenObject.server) else { return false }
-        if let (_, contract) = platforms.first(where: { platformMatches($0.key, server: tokenObject.server) }) {
+        if let (_, maybeContractValue) = platforms.first(where: { platformMatches($0.key, server: tokenObject.server) }) {
+            func maybeAddressValue(from str: String) -> AlphaWallet.Address? {
+                let rawValue = str.trimmed
+                if rawValue.isEmpty {
+                    //CoinGecko returns nullAddress as the value (contract) in `platforms` for tokens is sometimes an empty string: `"platforms" : { "ethereum" : "" }`, so we use the 0x0..0 address to represent them
+                    return Constants.nullAddress
+                } else if let value = AlphaWallet.Address(string: rawValue) {
```
Comment (hboon) [inline AlphaWallet/Core/CoinTicker/CoinTickersFetcher.swift:289]: Why is it faster if we delay `AlphaWallet.Address(string:)` till we do it here? Wouldn't we potentially be doing it more often for the same `rawValue`?

Comment (oa-s) [reply AlphaWallet/Core/CoinTicker/CoinTickersFetcher.swift:289]: its faster as we filter out a lot of objects when do:
`platformMatches($0.key, server: tokenObject.server)` and we don't create not needed addresses

**Comment by oa-s** [comment]
### Code execution time

| Before PR | After PR  |
| -------- | ---------- |
|Decoding takes: **0.7874749898910522** _seconds_|Decoding takes: **26.22158193588257** _seconds_|

**Comment by oa-s** [comment]
Execution time measured with: `CFAbsoluteTimeGetCurrent()` [CFAbsoluteTimeGetCurrent](https://www.hackingwithswift.com/example-code/system/measuring-execution-speed-using-cfabsolutetimegetcurrent)

**Comment by hboon** [comment]
Ok. But maybe the table is the wrong way around? We want after PR to be faster :)

**Review by hboon — APPROVED** [review]

**Pull request merged**